### PR TITLE
Add validation on expire date

### PR DESCRIPTION
**Pull request by pantelm**
Validate if the Credit Card is not expired.

### CODE REVIEW

File: Validator/Constraints/CreditCardDate.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    public $minMessage = 'validator.credit_card.invalidDate';
+    public $invalidMessage = 'validator.credit_card.invalidDate';
+    public $min;
+
+    public function __construct($options = null)
+    {
+        parent::__construct($options);
+
+        if (null === $this->min) {
+            throw new MissingOptionsException('Either option "min" or "max" must be given for constraint ' . __CLASS__, array('min', 'max'));
```
Comment (makasim) [inline Validator/Constraints/CreditCardDate.php]: max? is it used?

**Comment by makasim** [comment]
Cannot really get a use case for this validator, Could you please explain what it solves?

**Comment by pantelm** [comment]
If the user enters a date in the past, the validator will throw an error without requesting the payment server.

File: Validator/Constraints/CreditCardDate.php (hunk 1 above)
Comment (pantelm) [inline Validator/Constraints/CreditCardDate.php]: No, max is not used. It's an error on copy/paste

Comment (makasim) [inline Validator/Constraints/CreditCardDate.php]: lets remove it.

**Comment by makasim** [comment]
Got it, Could you please move it to the bridge? https://github.com/Payum/Payum/tree/master/src/Payum/Core/Bridge/Symfony

**Comment by pantelm** [comment]
I'll try but the validation constraint are define in the payum-bundle

**Comment by makasim** [comment]
Yes you are right. The classes should be kept in the bridge but registered in the bundle as services

**Comment by makasim** [comment]
Then it could be reused in silex for example.

**Pull request closed without merging**